tests/fuzz_parser.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/python3
+
+from decimal import InvalidOperation
+import atheris
+import sys
+
+
+with atheris.instrument_imports():
+    from dateutil.parser import parse, ParserError
+
+
+def TestOneInput(data):
+    fdp = atheris.FuzzedDataProvider(data)
+    try:
+        parse(fdp.ConsumeString(len(data)))
+    except (ParserError, OverflowError, InvalidOperation):
```

Comment:
I wasn't sure if I should include `InvalidOperation` in the catch list. Without it the fuzzer crashes very quickly due to #1199.